job_bundles/README.md

```diff
@@ -0,0 +1,315 @@
+# AWS Deadline Cloud Job Bundles
+
+Job Bundles are one of the tools that you can use to define your Jobs for AWS Deadline Cloud. They encapsulate
+an [Open Job Description Job Template](https://github.com/OpenJobDescription/openjd-specifications/wiki) into a directory
+with additional information such the files and directories that your Jobs need for
+Deadline Cloud's [Job Attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html)
+feature. The [Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud) provides ways for you to use a Job Bundle
+to submit Jobs to your Deadline Cloud Queues to be run.
+
+We recommend developing your own Job Bundle(s), either by-hand or programatically, to benefit from their
+integration with Job Attachments and an intuitive graphical Job submission interface. For example, after 
+[configuring the Deadline Cloud CLI](https://github.com/aws-deadline/deadline-cloud/blob/mainline/README.md#configuration) 
+you can run it with the Blender sample in this samples directory (`deadline bundle gui-submit blender_render/`) to see:
+
+![UI Shared Settings](../.images/blender_submit_shared_settings.png) ![UI Job Settings](../.images/blender_submit_job_settings.png) ![UI Job Attachments](../.images/blender_submit_job_attachments.png) 
+
+where the contents of the Job-specific settings panel are automatically generated using the `userInterface` properties of Job Parameters
+[defined in the Job Template](https://github.com/aws-deadline/deadline-cloud-samples/blob/bdd5ff5ea29eb7457c9a78ba39166b891b79151e/job_bundles/blender_render/template.yaml#L11-L19) within 
+your Job Bundle, and the [Queue Environments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html)
+defined on the Queue that you are submitting to.
+
+If you prefer an alternative to a UI-based workflow for your Job Bundles, then you can submit this Job Bundle with the command
+`deadline bundle submit --name Demo -p BlenderSceneFile=<location-of-your-scene-file> -p OutputDir=<file-path-for-job-outputs> blender_render/`
+or use the `deadline.client.api.create_job_from_job_bundle` function in the [`deadline` Python package](https://github.com/aws-deadline/deadline-cloud).
+You can also develop your Job as a Job Template and use the
```

Comment:
I think it's better to exclude this sentence. For example, things like passing job parameters and likely other things are trickier this way. We can create an advanced documentation page that covers all those details later.